flow-sample-basic/src/main/java/flow/sample/basic/Screens.java

```diff
@@ -0,0 +1,26 @@
+package flow.sample.basic;
+
+import android.content.Context;
+
+final class Screens {
+  private static final String SCREEN = "SERVICE_SCREEN";
+
+  public static <T> T getScreen(Context context) {
+    //noinspection unchecked,ResourceType
+    return (T) context.getSystemService(SCREEN);
+  }
+
+  static final class ContextWrapper extends android.content.ContextWrapper {
```

Comment:
Do you really want sample code that makes a context w/o the usual layout inflater boilerplate? Does the sample actually need this getScreen() method?